packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -637,6 +639,8 @@ class _IOSSimulatorLogReader extends DeviceLogReader {
     _systemProcess.stdout.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
     _systemProcess.stderr.transform(UTF8.decoder).transform(const LineSplitter()).listen(_onSystemLine);
 
+    // Callback hookups' Futures are uninteresting.
+    // ignore: unawaited_futures
     _deviceProcess.exitCode.whenComplete(() {
```

Comment:
though actually, why don't we want to await the exit code? Don't we want to make sure things are cleaned up at some point?